scss/_settings_colors.scss

```diff
@@ -3,6 +3,7 @@
 $color-transparent:      transparent !default;
 
 $color-brand:            #333 !default;
+$color-accent:        #007aa6 !default;
```

Comment:
If I am not mistaken `$color-accent` was to add personality strip to the brochure theme. Why are we using that variable here? I thought that we were going to just uncouple `$color-highlight` from '$color-information' and renaming `$color-highlight` to `$color-link`

https://docs.google.com/document/d/1Tx9pbW_nKSNEtDB_hziDNIZZl-oJMKGZc-jOIUv_PxI/edit#heading=h.9n8tll52kpfr